src/smart-components/workflow/workflows.js

```diff
@@ -54,26 +58,37 @@ const Workflows = ({ fetchRbacGroups, fetchWorkflows, workflows, pagination, his
           variant="primary"
           aria-label="Create Workflow"
         >
-Create Workflow
+          Create Workflow
         </Button>
       </Link>
     </ToolbarItem>
   </ToolbarGroup>;
 
   return (
-    <TableToolbarView
-      data={ workflows }
-      createInitialRows={ createInitialRows }
-      columns={ columns }
-      fetchData={ fetchData }
-      request={ fetchWorkflows }
-      routes={ routes }
-      actionResolver={ actionResolver }
-      titlePlural="Workflows"
-      titleSingular="Workflow"
-      pagination={ pagination }
-      toolbarButtons={ toolbarButtons }
-    />
+    <Fragment>
```

Comment:
This seems to me as a duplicate of this component: https://github.com/ManageIQ/approval-ui/pull/26/files#diff-97b72f94af4995a70aa8bb3c79802026R60

Can you make shared component for it?

Reply:
Moved to common component and update header styling.